tests/test_pdir_format.py

```diff
@@ -54,6 +55,8 @@ def what(self):
     print(result)  # TODO: add real test.
 
 
+@pytest.mark.xfail(sys.version_info[:2] not in ((2, 7), (3, 6)),
+                   reason='only fail with py27 or py36')
```

Comment:
Wrong wording?

Reply:
I'm indifferent to the wording, so whatever you like.  My intent was "on py27 or py36 this can fail the test suite, but otherwise it's an expected fail".

FYI I have checked the box for "allow edits from maintainers", so you can push commits to my PR branch [carlwgeorge/pdir2#xfail_test_pdir_class](https://github.com/carlwgeorge/pdir2/tree/xfail_test_pdir_class) to rephrase it however you see fit.

Reply:
I'm not using my working computer. I would go with
`not intended to be tested under Python %s" % sys.version[:3]`
or similar thing.

Reply:
Updated as requested in a new amended commit.